Gordon360/Services/ProfileService.cs

```diff
@@ -323,6 +326,47 @@ public async Task<StudentProfileViewModel> UpdateMobilePhoneNumberAsync(string u
             return profile;
         }
 
+        /// <summary>
+        /// office location setting
+        /// </summary>
+        /// <param name="username"> The username for the user whose office location is to be updated </param>
+        /// <param name="newBuilding">The new building location to update the user's office location to</param> 
+        /// <param name="newRoom">The new room to update the user's office room to</param>
+        public async Task<FacultyStaffProfileViewModel> UpdateOfficeLocationAsync(string username, string newBuilding, string newRoom)
```

Comment:
```suggestion
        /// <param name="newRoom">The new room to update the user's office room to</param>
        public async Task<FacultyStaffProfileViewModel> UpdateOfficeLocationAsync(string username, string newBuilding, string newRoom)
      
```
We have a return value, it would be beneficial to know what you're returning if anything

Reply:
Sounds good! Just fixed it